odata-url-conventions/meta.yaml

```diff
@@ -6,6 +6,8 @@ previousStage: csd01
 pubdate: 28 February 2024
 pubdateISO: '2024-02-28'
 copyright: © OASIS Open 2024
+archivename: odata-v4.02-csd02
+partname: part2-url-conventions
```

Comment:
Should we remove `filename` and use these two instead?

Reply:
And needs more care in maintaining meta.yaml

Reply:
We use `filename` already in our markdown sources. OData-Protocol currently has
```yaml
filename: odata-v4.02-csd02-part1-protocol
archivename: odata-v4.02-csd02
partname: part1-protocol
```
What you propose is
```js
filename = archivename + (partname ?
  "-" + partname : "")
```
But this dynamic computation would have to be done by every consumer of the meta.yaml file.

Reply:
We could split the filename at the hyphen that is followed by "part" and avoid `archivename` and `partname`.